common/lib/populate-resources.js

```diff
@@ -0,0 +1,36 @@
+const moveService = require('../services/move')
+const personEscortRecordService = require('../services/person-escort-record')
+const referenceDataService = require('../services/reference-data')
+
+const findUnpopulatedResources = require('./find-unpopulated-resources')
+
+const lookupMethods = {
+  locations: referenceDataService.getLocationById,
```

Comment:
Would be even nicer, since the req knows the resources are to be populated for the api-client or the services to do it. But without circularising the api-client and/or the services, I can't see how to do it.